### PR TITLE
updated using sass

### PR DESCRIPTION
**Pull request by ashleymcnamara**

### CODE REVIEW

File: start-devel.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@
 # First you must fill you github application credentials below.
 #
 
-export GITHUB_CLIENT_ID=$1
-export GITHUB_CLIENT_SECRET=$2
+export GITHUB_CLIENT_ID=$4adb262334587c3bd31c
+export GITHUB_CLIENT_SECRET=$65bb8720dd356b0d7437b07e67a7246f9f18664c
 export OO_INDEX_GITHUB_USERNAME=${3:-openshift}
```
Comment (caruccio) [inline start-devel.sh:9]: I guess that you want to do this instead:

```
$ ./start-devel.sh 4adb262334587c3bd31c 65bb8720dd356b0d7437b07e67a7246f9f18664c
```

File: wsgi/myflaskapp.py [hunk 2]
```diff
@@ -47,16 +47,16 @@ class OOIndexError(Exception):
 Funnel(app)
 
 try:
```
Comment (caruccio) [inline wsgi/myflaskapp.py:49]: Same here. Those files don't need to be changes. You can pass all necessary data through parameters or environment variables.
For example, to change GITHUB_CALLBACK_URL, while using you own Github credentials, do like this:

```
$ export GITHUB_CALLBACK_URL=myooindexapp-mynamespace.rhcloud.com
$ ./start-devel.sh  MY_GITHUB_CLIENT_ID  MY_GITHUB_CLIENT_SECRET 
```

**Comment by caruccio** [comment]
Hi @ashleymcnamara.
Are the files `*/config.codekit` and `drupal-x.x.tar.gz` really necessary?

BTW, thanks for the huge commit!
Do you have a live running app with these changes?

File: wsgi/myflaskapp.py (hunk 2 above)
Comment (ashleymcnamara) [inline wsgi/myflaskapp.py:49]: Shoot! I forgot to push. I'm sorry. :/ I'll fix that right now.

Thank You
.  .  .  .  .  .  .  .  .  .  .  . 
Ashley McNamara  
Website | Twitter | Facebook

**Comment by ashleymcnamara** [comment]
Oh wait! You got it? Whew, I thought I forgot to push. I didn't deploy it
but I was live coding it.

On Sun, Apr 20, 2014 at 7:24 PM, Mateus Caruccio
notifications@github.comwrote:

> Hi @ashleymcnamara https://github.com/ashleymcnamara.
> Are the files */config.codekit and drupal-x.x.tar.gz really necessary?
> 
> BTW, thanks for the huge commit!
> Do you have a live running app with these changes?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/openshift/oo-index/pull/51#issuecomment-40911968
> .

## 

Thank You
.  .  .  .  .  .  .  .  .  .  .  .
Ashley McNamara
Website http://www.ashleymcnamara.com |
Twitterhttps://twitter.com/ashleymcnamara|
Facebook  https://www.facebook.com/ashleymcnamaraphotography

**Comment by ashleymcnamara** [comment]
I need to update my gitignore file. I am learning.

**Comment by ashleymcnamara** [comment]
I am new to this, I just updated things so I should be good to go now? Not sure. Let me know.

File: wsgi/myflaskapp.py (hunk 2 above)
Comment (ashleymcnamara) [inline wsgi/myflaskapp.py:49]: Also, thanks for the advice. I need all the advice I can get. :)

**Comment by caruccio** [comment]
Yep, just commit and push again. The PR will reflect the branch's current state.

File: start-devel.sh [hunk 3]
```diff
@@ -4,8 +4,8 @@
 # First you must fill you github application credentials below.
 #
 
-export GITHUB_CLIENT_ID=$1
-export GITHUB_CLIENT_SECRET=$2
+export GITHUB_CLIENT_ID=
+export GITHUB_CLIENT_SECRET=
 export OO_INDEX_GITHUB_USERNAME=${3:-openshift}
```
Comment (caruccio) [inline start-devel.sh:9]: Missing parameters $1 and $2

Comment (ashleymcnamara) [inline start-devel.sh:9]: I changed it and pushed it again. I super apologize if I am doing all of this wrong? Thanks for being patient.

**Comment by dmueller2001** [comment]
Ash,  I am going to close this for now as it still is throwing errors when I try and deploy from your repo. Ping me when you get a chance and we can sync up

Here's a snippet: 

clean-css@2.1.8 node_modules/clean-css
└── commander@2.1.0
Missing config: '' (Please fix)
Fixing URL's in css/normalize.css
Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    manager.run()
  File "/var/lib/openshift/5359ab1ee0b8cdc1de0006cd/python/virtenv/lib/python2.7/site-packages/Flask_Script-0.6.7-py2.7.egg/flask_script/**init**.py", line 423, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "/var/lib/openshift/5359ab1ee0b8cdc1de0006cd/python/virtenv/lib/python2.7/site-packages/Flask_Script-0.6.7-py2.7.egg/flask_script/**init**.py", line 402, in handle
    return handle(app, _positional_args, *_kwargs)
  File "/var/lib/openshift/5359ab1ee0b8cdc1de0006cd/python/virtenv/lib/python2.7/site-packages/Flask_Script-0.6.7-py2.7.egg/flask_script/commands.py", line 145, in handle
    return self.run(_args, *_kwargs)
  File "/var/lib/openshift/5359ab1ee0b8cdc1de0006cd/python/virtenv/lib/python2.7/site-packages/Flask_Funnel-0.1.10-py2.7.egg/flask_funnel/manager.py", line 156, in bundle_assets
    processed = preprocess_file(fn, compressed_file)
  File "/var/lib/openshift/5359ab1ee0b8cdc1de0006cd/python/virtenv/lib/python2.7/site-packages/Flask_Funnel-0.1.10-py2.7.egg/flask_funnel/manager.py", line 110, in preprocess_file
    filename = fix_urls(filename, compressed_file)
  File "/var/lib/openshift/5359ab1ee0b8cdc1de0006cd/python/virtenv/lib/python2.7/site-packages/Flask_Funnel-0.1.10-py2.7.egg/flask_funnel/manager.py", line 44, in fix_urls
    with open(get_path(filename), 'r') as css_in:
IOError: [Errno 2] No such file or directory: '/var/lib/openshift/5359ab1ee0b8cdc1de0006cd/app-root/runtime/repo/wsgi/static/css/normalize.css'
An error occurred executing 'gear postreceive' (exit code: 1)
Error message: CLIENT_ERROR: Failed to execute action hook 'build' for 5359ab1ee0b8cdc1de0006cd application indexjsond

**Pull request closed without merging**